src/als/ui/prefs_ui.ui

```diff
@@ -91,12 +91,111 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <widget class="QGroupBox" name="groupBox_2">
+     <property name="title">
+      <string>Pre-Processing</string>
+     </property>
+     <layout class="QHBoxLayout" name="horizontalLayout_3">
+      <item>
+       <layout class="QGridLayout" name="gridLayout">
+        <item row="0" column="0">
+          <widget class="QCheckBox" name="chk_use_dark">
+           <property name="text">
+            <string>&amp;Use master dark</string>
+           </property>
+          </widget>
+        </item>
+        <item row="1" column="0">
+         <widget class="QLabel" name="lbl_dark_folder">
+          <property name="text">
+           <string>Master &amp;dark folder :</string>
```

Comment:
this 'd' accelerator is already used (By 'Debug logs')

Reply:
Damned, didn't see it :P

Reply:
Done